build/config/SignToolData.json

```diff
@@ -105,7 +105,9 @@
       "values": [
         "Dlls\\ServicesTestUtilities\\Roslyn.Services.Test.Utilities.dll",
         "Dlls\\TestUtilities\\net461\\Roslyn.Test.Utilities.dll",
-        "Dlls\\VisualStudioIntegrationTestUtilities\\Microsoft.VisualStudio.IntegrationTest.Utilities.dll"
+        "Dlls\\VisualStudioIntegrationTestUtilities\\Microsoft.VisualStudio.IntegrationTest.Utilities.dll",
+        "Dlls\\Diagnostics\\Roslyn.Hosting.Diagnostics.dll",
```

Comment:
@tannergooding @jaredpar this "Roslyn.Hosting.Diagnostics.dll" appears in above section 
{ 
   "certificate": "MicrosoftSHA2",
  "strongName": "MsSharedLib72",

so not sure whether it is okay to list it twice here with different strong name, or the existing one is good enough?

Reply:
@tannergooding how about this? should dll only appear once in this file? what if same dll needs to be included in 2 different vsix? does it need to be all signed with same keys for same vsix?

Reply:
okay. build failed with this
16:33:25 Duplicate file entry: Dlls\Diagnostics\Roslyn.Hosting.Diagnostics.dll

so, apparently duplicated file entry is wrong. I will just remove the new entry I added